packages/react/ds/src/forms/form-field/form-field.tsx

```diff
@@ -102,7 +96,7 @@ const FormFieldBase = ({
 
   return (
     <fieldset
-      className={cn({ 'gi-error-state': !!error }, className)}
+      className={cn({ 'gi-error-state': !!error }, props.className)}
       {...props}
```

Comment:
it needs to extract className from props otherwise if className is set 'gi-error-state' will be replaced.